src/main/java/com/opencart/steps/CurrencyPageBL.java

```diff
@@ -0,0 +1,102 @@
+package com.opencart.steps;
+
+import com.opencart.pages.CurrencyPage;
+import com.opencart.util.DriverUtils;
+import org.testng.Assert;
+
+public class CurrencyPageBL {
+
+    private CurrencyPage currencyPage;
+
+    public CurrencyPageBL() {
+        currencyPage = new CurrencyPage();
+    }
+
+    public AddNewCurrencyPageBL getToAddingNewCurrency() {
+        clickOnAddNewCurrencyButton();
+        return new AddNewCurrencyPageBL();
+    }
+
+    public CurrencyPageBL deleteCurrency() {
+        sortCurrencyByLastUpdateFromOldest();
+        clickOnInputCheckbox();
+        clickOnDeleteCurrencyButton();
+        new DriverUtils().clickAcceptOnPopUpWindow();
+        return new CurrencyPageBL();
+    }
```

Comment:
Maybe it is better to remove currency by title? - it will make your method more flexible